servers/zms/src/main/java/com/yahoo/athenz/zms/store/jdbc/JDBCConnection.java

```diff
@@ -666,13 +668,27 @@ PreparedStatement prepareDomainScanStatement(String prefix, long modifiedSince)
             }
         } else if (modifiedSince != 0) {
             ps = con.prepareStatement(SQL_LIST_DOMAIN_MODIFIED);
-            Calendar cal = Calendar.getInstance(TimeZone.getTimeZone("GMT"));
             ps.setTimestamp(1, new java.sql.Timestamp(modifiedSince), cal);
         } else {
             ps = con.prepareStatement(SQL_LIST_DOMAIN);
         }
         return ps;
     }
+
+    Calendar getCalendarInstance() {
+
+        Boolean useLocalTimeZone = Boolean.parseBoolean(
```

Comment:
we should not fetch the property and parse it every time. this should be a static member of the class  so we only do it once and never again.